### ISSUE TITLE
Tree structure

### ISSUE THREAD

**Issue by alabotski** [GitFiend/Support#139]
You can make it possible to display files as a tree instead of a list

![image](https://user-images.githubusercontent.com/439337/197353268-9533e263-570d-4bf5-91e8-2dff74ccf37a.png)

**Comment by GitFiend** [comment]
Yeah, I can look into this. Being able to select/unselect a folder would be handy.
